float32[] time
float32[] effort
float32[] position
float32[] velocity
uint8 dir
uint8 UP=0
uint8 DOWN=1
